integration/README.md

```diff
@@ -0,0 +1 @@
+## Interop testing
```

Comment:
lets add instructions on running here, which ideally should just be "run make test"

Reply:
Done, thank you!